helper-cli/src/main/kotlin/commands/GenerateRuleViolationResolutionsCommand.kt

```diff
@@ -58,16 +60,19 @@ internal class GenerateRuleViolationResolutionsCommand : CliktCommand(
 
     private val severity by option(
         "--severity",
-        help = "Only consider violations of the given severity. Allowed values: ERROR|WARNING|HINT."
+        help = "Only consider violations of the given severities, specified as comma separated values. Allowed " +
+                "values: ERROR|WARNING|HINT."
     ).enum<Severity>()
+        .split(",")
+        .default(listOf(*Severity.values()))
```

Comment:
More Kotlin-ish: `.default(enumValues<Severity>().asList())`

Reply:
Also, this could be a one-liner.

Reply:
Done x2.